### DISCUSSION TITLE
failed to register layer: unlinkat /usr/share/doc/intel-media-va-driver: invalid argument

### DISCUSSION THREAD

**Discussion by greggft** [koush/scrypted#986] [Q&A]
running under container in Proxmox
every pull does *not* "keep/stay", it always pulls every time.  My other dockers pull full once and stays
after every pull or every run I get this error
failed to register layer: unlinkat /usr/share/doc/intel-media-va-driver: invalid argument
I even went ahead and installed via apt intel-media-va-driver and rebooted the container and still get the error
scripted@scr-01:~/.scrypted$ uname -a
Linux scr-01 6.2.16-3-pve #1 SMP PREEMPT_DYNAMIC PVE 6.2.16-3 (2023-06-17T05:58Z) x86_64 x86_64 x86_64 GNU/Linux
scripted@scr-01:~/.scrypted$ docker version
Client: Docker Engine - Community
 Version:           24.0.5
 API version:       1.43
 Go version:        go1.20.6
 Git commit:        ced0996
 Built:             Fri Jul 21 20:35:14 2023
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          24.0.5
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.6
  Git commit:       a61e2b4
  Built:            Fri Jul 21 20:35:14 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.21
  GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
 runc:
  Version:          1.1.7
  GitCommit:        v1.1.7-0-g860f061
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
scripted@scr-01:~/.scrypted$ 

Please help.  I have the same "docker environment" in another container running Wyse-Bridge for my Wyze cameras
Thank you

UPDATE
First it appears according to what I read at Proxmox they don't suggest docker in containers,
Second I found someone fixed this with changing the uid and gid range to a *huge* range via /etc/subuid and /etc/subgid and reboot

Running in a vm really isn't going to work for me with my limited computer I am running on
I changed the range as suggested and rebooted and still no good.
Something is possibly wrong with my setup because again the pull does not stick and the syslog shows after everything is downloaded it "cleans" each download, yet it still "appears" in the docker image area.  I might just rebuild my environment

I will update after I am done

UPDATE #2
No difference 
Same problems
Then tried rootless, but couldn't get it to work....

UPDATE #3 
I guess I will get rid of the docker setup and try another method to run Scrypted
Not what I want to do.....

**Comment by Pinsyd** [comment]
Like the original OP, I too was having no luck  - same Intel error on AMD hardware in LXC and VM w/ docker and no love for apt install because deb repo did not support amd64.  There is a Proxmox LXC installer for scrypted and it worked just fine!

[https://tteck.github.io/Proxmox/](https://tteck.github.io/Proxmox/) -> Automation -> Scrypted

takes over an hour to install but I could login and add cameras once it was done.  Hope this helps!
